api/types/types.go

```diff
@@ -220,6 +217,15 @@ type Info struct {
 	Warnings []string
 }
 
+// ContainerStateInfo holds the total number of containers, and the number of
+// containers for each state (running, paused, and stopped).
+type ContainerStateInfo struct {
```

Comment:
I'm not sure I understand what this specific part of this change does for us? :see_no_evil:

(As far as I can tell, it just makes everywhere we use it more verbose :sweat_smile:)

Reply:
Ah, yes, that's fair. I think I was playing with some variations here (considering to leave things `nil` if we don't set them.

But let me drop that commit, and keep that for when we're actually gonna make things more granular

Reply:
OK; removed the commit 👍